metadata-models/src/main/pegasus/com/linkedin/datajob/DataJobInfo.pdl

```diff
@@ -72,4 +73,15 @@ record DataJobInfo includes CustomProperties, ExternalReference {
    */
   @deprecated = "Use Data Process Instance model, instead"
   status: optional JobStatus
+
+  /**
+   * Environment for this job
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
```

Comment:
This should be keyword field, even if this was done else where. There is no need to try to tokenize, stem, etc. This should be exact match or no match especially since they are often abbreviations of real words.